docker/db-migration/migrate.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+cd /liquibase
+
+DB_NAME="${KILDA_ORIENTDB_DB_NAME:-kilda}"
+DB_HOST="${KILDA_ORIENTDB_HOST:-odb3.pendev}"
+
+echo "Apply main migrations set (root.yaml)"
+./liquibase --username="${KILDA_ORIENTDB_USER:-kilda}" --password="${KILDA_ORIENTDB_PASSWORD:-kilda}" \
+  --url=jdbc:orient:remote:"${DB_HOST}/${DB_NAME}" \
+  --driver=com.orientechnologies.orient.jdbc.OrientJdbcDriver --changeLogFile=migrations/root.yaml update
```

Comment:
Let's consider tagging as the rollback strategy. See https://docs.liquibase.com/change-types/community/tag-database.html

Reply:
ok. There is no actual migration here, so I will try to describe it in README.